spec/requests/citizens/dependants_spec.rb

```diff
@@ -41,8 +41,10 @@
     end
 
     it 'redirects to the dependant relationship page if the dependant is more than 15 years old' do
+      dependant_id = SecureRandom.uuid
+      allow_any_instance_of(Dependant).to receive(:id).and_return(dependant_id)
```

Comment:
I don't think you need to stub here and can avoid using `allow_any_instance_of`. Instead identify the dependent that was just created and use its id. So:
```ruby
let(:latest_dependant) { Dependant.order(:created_at).last }

it 'redirects to the dependant relationship page if the dependant is more than 15 years old' do
  subject
  expect(response).to redirect_to(citizens_dependant_relationship_path(latest_dependant.id))
end
```

Reply:
Good idea 👍